app/models/vm_migrate_workflow.rb

```diff
@@ -22,6 +22,8 @@ def get_source_and_targets(refresh = false)
     return @target_resource = {} if ems.length != 1
 
     result = {:ems => ci_to_hash_struct(ems.first)}
+    @manager = ems.first
```

Comment:
I think you can remove this here and inline that below in `field_unsupported`
Because like this its adding a side effect to this method